miner/miner.go

```diff
@@ -144,7 +144,29 @@ func (m *Miner) Start(_ context.Context) error {
 		return fmt.Errorf("miner already started")
 	}
 	m.stop = make(chan struct{})
-	go m.mine(context.TODO())
+
+	// block production only enabled in lotus-miner for
+	// FilecoinEC consensus.
+	if build.Consensus == build.FilecoinEC {
+		go m.mine(context.TODO())
```

Comment:
Can we use a well-defined context here?

Reply:
This comes from Lotus, I haven´t changed anything (apart from adding the if). If you look at the top-level function, it disregards the context. I rather leave it as-is for now so we don´t  break anything. I don´t know the reason why the Lotus team decided to do it that way.